src/mlpack/tests/ann/layer/layer_norm.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file tests/layer/layer_norm.cpp
+ * @author Marcus Edel
+ * @author Praveen Ch
```

Comment:
Feel free to add your name here (and elsewhere!).

Reply:
Will do.  What's your guidelines on what contributions get an @author mention?

Reply:
We don't really have any particular guidelines, so "has modified at least one character in the file" seems good to me :smile: definitely you have done a ton of work on these two layers and that would meet any reasonable bar. :+1:

Reply:
👍